src/Microsoft.CodeAnalysis.Testing/Microsoft.CodeAnalysis.Analyzer.Testing/AnalyzerTest`1.cs

```diff
@@ -497,6 +497,13 @@ private async Task VerifyGeneratedCodeDiagnosticsAsync(ImmutableArray<Diagnostic
 
         private async Task VerifySuppressionDiagnosticsAsync(ImmutableArray<DiagnosticAnalyzer> analyzers, (string filename, SourceText content)[] sources, EvaluatedProjectState primaryProject, ImmutableArray<EvaluatedProjectState> additionalProjects, DiagnosticResult[] expected, IVerifier verifier, CancellationToken cancellationToken)
         {
+            if (LightupCompilationWithAnalyzers.IsTestingDiagnosticSuppressors(analyzers))
+            {
+                // Actually the way suppressed diagnostics are reported in CA framework 4.x conflicts with the way VerifySuppressionDiagnosticsAsync is implemented, so this will always fail.
+                // Anyhow if we test diagnostic suppressors, this check is not required, since it tests suppressing issues via source code by adding #pragma warning disable
+                return;
+            }
```

Comment:
💭 I would not expect this to be necessary. Need to think about this one a bit more.

Reply:
To me it just made no sense to test if you can suppress a suppressed message